### PR TITLE
Added sort in url and saving sort selection in local cache

### PR DESCRIPTION
**Pull request by seansaleh**
Updated tablesorter
Also snuck in a change to make column titles hover with a cursor and the download button to have alt-text

### CODE REVIEW

**Comment by seansaleh** [comment]
@gerow I can haz review?

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@ DATA_DIR    = data
 
 DATA_FILES  = $(DATA_DIR)/templates/dir_listing.html
 DATA_FILES += $(DATA_DIR)/version_hash
-DATA_FILES += $(DATA_DIR)/static/js/jquery.tablesorter.min.js
+DATA_FILES += $(DATA_DIR)/static/js/jquery.tablesorter.combined.min.js
```
Comment (gerow) [inline Makefile]: Why do you need this combined thing?

Comment (seansaleh) [inline Makefile]: This is the latest version of TableSorter built with its extensions saveSort and sort2Hash
I meant to add a note somewhere, but didn't (and not sure where would be best to)
https://github.com/Mottie/tablesorter/wiki/Customize
Those are the docs on how to minifiy/uglify a version of tablesorter with specific extensions/widgets
I didn't want to include all the widgets that we are not using that tablesorter has

**Comment by gerow** [comment]
Ok. I haven't tested this myself... but nothing here looks bad to me. You can go ahead and merge it yourself since you should have commit rights.

**Pull request merged**